### PR TITLE
Add page Pet Profile

### LINKED ISSUE #27: Implement Dog Card page

### PR DESCRIPTION
**Pull request by okhabr**
- add pet component;
 - add complementing components to the pet component
   1.additionalInfo
   2.characteristics
   3.observations
   4.circles
   5.logo

### CODE REVIEW

**Review by Pashkaua — APPROVED** [review]
Look nice !

File: src/components/pet/pet.component.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  };
+}
+export const Pet = (props: PetProps): React.ReactElement => {
+  const petId: number = props.match.params.id;
+  const [imgUrl, setImgUrl] = React.useState<string>('');
+  const [breed, setBreed] = React.useState<string>('');
+  const [characteristics, setCharacteristics] = React.useState();
+  const [observations, setObservations] = React.useState();
+  const [additionalInfo, setAdditionalInfo] = React.useState();
+  const [loading, setLoading] = React.useState<boolean>(true);
```
Comment (ivan-bonk) [inline src/components/pet/pet.component.tsx:23]: Why don't you destructure useState into import like useEffect. 
So you avoid repeating React. in each line

File: src/components/pet/pet.component.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  async function getPet(id: number) {
+    const dogProfile = await fetch(`http://localhost:5000/pets/${id}`);
+    if (dogProfile.ok) {
+      return await dogProfile.json();
+    } else {
+      return Promise.reject();
+    }
+  }
```
Comment (ivan-bonk) [inline src/components/pet/pet.component.tsx:32]: I thing it would be better to use Saga and write data to global State using Redux

**Review by ivan-bonk — APPROVED** [review]
Good job!
Your code looks very structured and easy to read and understand.

File: src/components/pet/pet.component.tsx (hunk 1 above)
Comment (okhabr) [reply src/components/pet/pet.component.tsx:23]: Good point, thanks. Corrected

File: src/components/pet/pet.component.tsx (hunk 2 above)
Comment (okhabr) [reply src/components/pet/pet.component.tsx:32]: Not sure if it is necessary to put this data in the global state, as it is not expected to be reused somewhere else in the application. I`ll seek Roma for guidance and come back to this. Thanks for the note

**Review by Danik1702 — APPROVED** [review]
Good!

**Comment by okhabr** [comment]
close #37

**Pull request closed without merging**